src/aten/BatchNorm.cpp

```diff
@@ -0,0 +1,353 @@
+#include <ATen/ATen.h>
+#include <ATen/AccumulateType.h>
+#include <ATen/XPUNativeFunctions.h>
+#include <ATen/core/Tensor.h>
+#include <ATen/core/op_registration/adaption.h>
+#include <aten/sycl/BatchNormKernels.h>
+
+namespace at {
+
+::std::tuple<Tensor, Tensor> XPUNativeFunctions::batch_norm_stats(
```

Comment:
Can't we use global namespace for `std`? I think no one define an ambiguous namespace under `at`.